# sets the angle of rotating table
float64 position
---
